README.md

```diff
@@ -19,24 +19,34 @@ Opni currently features log anomaly detection for Kubernetes.
   * Ability to setup & send alerts (slack/email/etc) based on Opni log insights
 
 ----
-#### Try it out!
+### Install Opni in your Kubernetes cluster:
+* Download the `opnictl` binary from the [latest release](https://github.com/rancher/opni/releases/tag/v0.1.2)
+* Install Opni using `opnictl`
+  ```
+  opnictl install
+  opnictl create demo
+  ```
+  * Will look at `~/.kube/config` for your cluster's kubectl configuration file
```

Comment:
This is not exactly correct - it uses the same logic that kubectl uses to determine which config and context to use. It would be more accurate to say "It will use your current kubeconfig context"